### PR TITLE
Fix Grid System docs

### PR DESCRIPTION
**Pull request by kohheepeace**
Fix Grid System docs mentioned at #33670

This is how the document will look after this PR.
2021-04-19_05h26_51

### CODE REVIEW

**Review by ffoodd — APPROVED** [review]
Thanks 👌

File: site/content/docs/5.0/layout/grid.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ Bootstrap's grid system uses a series of containers, rows, and columns to layout
 {{< example class="bd-example-row" >}}
 <div class="container">
   <div class="row">
-    <div class="col-sm">
-      One of three columns
+    <div class="col">
+      ⅓ column
```
Comment (mdo) [inline site/content/docs/5.0/layout/grid.md:21]:
```suggestion
      Column
```

File: site/content/docs/5.0/layout/grid.md [hunk 2]
```diff
@@ -17,14 +17,14 @@ Bootstrap's grid system uses a series of containers, rows, and columns to layout
...
     </div>
-    <div class="col-sm">
-      One of three columns
+    <div class="col">
+      ⅓ column
```
Comment (mdo) [inline site/content/docs/5.0/layout/grid.md:24]:
```suggestion
      Column
```

File: site/content/docs/5.0/layout/grid.md [hunk 3]
```diff
@@ -17,14 +17,14 @@ Bootstrap's grid system uses a series of containers, rows, and columns to layout
...
     </div>
-    <div class="col-sm">
-      One of three columns
+    <div class="col">
+      ⅓ column
```
Comment (mdo) [inline site/content/docs/5.0/layout/grid.md:27]:
```suggestion
      Column
```

**Review by mdo — APPROVED** [review]
Prefer not using the fractions here, so simplifying.

**Pull request merged**